src/org/recompile/mobile/PlatformPlayer.java

```diff
@@ -56,17 +56,17 @@ public PlatformPlayer(InputStream stream, String type)
 		}
 		else
 		{
-			if(type.equals("audio/midi") || type.equals("sp-midi") || type.equals("audio/spmidi"))
+			if(type.equals("audio/mid") || type.equals("Audio/midi") || type.equals("audio/midi") || type.equals("sp-midi") || type.equals("audio/spmidi"))
 			{
 				player = new midiPlayer(stream);
 			}
 			else
 			{
-				if(type.equals("audio/x-wav"))
+				if(type.equals("audio/x-wav") || type.equals("audio/X-wav") || type.equals("audio/wav"))

```

Comment:
I guess you could also use `equalsIgnoreCase()` I guess whatever way looks nicer.

Reply:
Good idea, i'll do it like that.